core/src/test/java/org/elasticsearch/index/mapper/MapperServiceTests.java

```diff
@@ -39,8 +29,17 @@
 import org.elasticsearch.index.mapper.NumberFieldMapper.NumberFieldType;
 import org.elasticsearch.test.ESSingleNodeTestCase;
 
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+import java.util.function.Function;
+
 import static org.hamcrest.CoreMatchers.containsString;
-import static org.hamcrest.Matchers.hasToString;
```

Comment:
I was reverting this bit, funnily enough this same import order change came in through another PR: https://github.com/elastic/elasticsearch/commit/b9600c78918579181164a03ba25579c8ea3b3f9e#diff-41da0c9cbefe797a78bab64005bf2b94 . :)